src/originalBudgetTrackingApp/bankScraper.ts

```diff
@@ -13,7 +13,7 @@ export const inputVendors = Object.keys(SCRAPERS)
 
 interface ScrapeParameters {
   companyId: AccountToScrapeConfig['key'];
-  credentials: AccountToScrapeConfig['credentials'];
+  credentials: AccountToScrapeConfig['loginFields'];
```

Comment:
```suggestion
  credentials: AccountToScrapeConfig.loginFields;
```

Without looking at the code, I think if you access to the property itself, it will force the type-checking, right?
I think if we had done it from the beginning, the bug would have been discovered during compilation, no?